hypothesis-python/tests/cover/test_validation.py

```diff
@@ -247,3 +248,10 @@ def test(x):
 
     with pytest.raises(InvalidArgument):
         test()
+
+
+def test_check_type_with_tuple_of_length_two():
+    # This test covers logic for length-two tuples that is essential on PY2,
+    # e.g. string_types (str, unicode) which are all length-one on Python 3.
+    check_type((int, str), 1)
+    check_type((int, str), "1")
```

Comment:
For full coverage you'll also need a failing case, e.g. using the pytest.raises context manager.

Reply:
Ok sure, I have added a pytest.raises to that test.